pkg/controller/installation/products/rhssouser/reconciler_test.go

```diff
@@ -413,6 +413,31 @@ func TestReconciler_fullReconcile(t *testing.T) {
 		},
 	}
 
+	installation := &v1alpha1.Installation{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:       "installation",
+			Namespace:  defaultRhssoNamespace,
+			Finalizers: []string{"finalizer.user-sso.integreatly.org"},
```

Comment:
do we want a test here that adds more than one finalizer? Perhaps it exists elsewhere?